modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,535 @@
+
+=== Some inline formatting, such as color and font size, can now be applied to list item elements
+
+In {productname} 6.2 list bullets and list numbers can be inline-formatted.
+
+If the contents of a list item are selected and inline formatting is applied — such as a change of color or change of font-size — this spot-formatting is also applied to the list item’s associated bullet or number.
+
+For more information see the xref:advlist.adoc#inline-formatting[Inline formatting of list elements documentation].
+
+=== The installed and available plugin lists in the Help dialog are now sorted alphabetically.
+
+Previously, when the *Help > Help* dialog was invoked, and the _Plugins_ list was selected, available Premium plugins were supposed to be presented in alphabetical order by name.
+
+This was done manually, however, and the manual presentation was not entirely alphabetical.
+
+Also, this manual sort did not take account of the selected UI language and the Premium plugin names were not, in the main, translated.
+
+With this update Premium plugin names are explicitly not translated. But they are, now, correctly sorted in alphabetical order by name.
+
+=== Alignment can now be applied to more embedded media element types.
+
+Embedded media elements could not, previously, be set to left, right, or center alignment.
+
+With this update they can. Selecting an embedded media element and clicking the left, right, or center alignment toolbar buttons, or selecting the appropriate option from the *Format > Align* sub-menu, sets the element to the chosen alignment.
+
+
+[[changes]]
+== Changes
+
+{productname} 6.2 also incorporates the following changes:
+
+=== The `@menubar-row-separator-color` oxide variable no longer affects the divider between the Menubar and Toolbar.
+
+Testing revealed circumstances in which an unnecessary border was added under the {productname} Menu bar.
+
+This update includes changes to the `@menubar-row-separator-color` oxide variable. It now only controls the color of the separator lines drawn in multiline Menubars.
+
+Consequently, unnecessary borders are no longer added under the {productname} menu bar.
+
+=== The `@toolbar-separator-color` oxide variable now affects the color of the separator between the Menu bar and Toolbar only.
+
+In concert with the testing above that discovered an unnecessary border occasionally being added under the {productname} Menu bar, the `@toolbar-separator-color` oxide variable was also changed.
+
+This variable now only affects the color of the separator between the {productname} Menu bar and the {productname} Toolbar.
+
+=== Available Premium plugins, which are listed by name in the Help dialog, are no longer translated.
+
+Previously, when the *Help > Help* dialog was invoked, and the _Plugins_ list was selected, available Premium plugins were supposed to be presented in alphabetical order by name.
```

Comment:
This section seems to be entirely duplicated from the above section in improvements. Is that intentional because it is both a change and an improvement?

Reply:
(discussion) de-duplicate.